NEWS.md

```diff
@@ -6,6 +6,8 @@
 
    Related to the above change, `navset_bar()` now defaults to using `underline = TRUE` so that both `page_navbar()` and `navset_bar()` use the same set of default `navbar_options()`.
 
+   In `navbar_options()`, `inverse` is replaced by `theme`, which takes values `"light"` (dark text on a **light** background), `"dark"` (light text on a **dark** background), or `"auto"` (follow page settings). (#1146)
```

Comment:
@gadenbuie couldn't we just change `inverse` to `theme` in the 1st bullet point (since `navbar_options()` is a new addition in this release)?

Reply:
I didn't change it because that sentence is talking about the direct arguments, which is still `inverse`. I could move this sentence, but I don't mind calling out the change on its own line.